File: graph.cpp
class Vertex{
  private: 
    vector<Edge> edges; 
  public:
    vector<Edge> getEdges(){
     return edges;
    }

  void  setEdges(List<Edge> e){
    edges = e;
  }

  void addEdge(Edge e){
    edges.push_back(e);
  }
};


public class Edge{
  private:
    Vertex u;
    Vertex v;
    int capacity;
  public:
    int getCapacity(){
      return capacity;
    }
};

File: graph.java
public class Vertex{
  private List<Edge> edges; 
  public  List<Edge> getEdges(){
    return edges;
  }

  public  void  setEdges(List<Edge> e){
    edges = e;
  }

  public void addEdge(Edge e){
    edges.add(e);
  }
}


public class Edge{
  private Vertex u;
  private Vertex v;
  private int capacity;
}
